MapboxCoreNavigation/RouteController.swift

```diff
@@ -130,6 +130,12 @@ open class RouteController: NSObject {
      */
     public var allowRecordedAudioFeedback = false
     
+    
+    /**
+     If true, spoken instructions for the route will be displayed on the map.
+     */
+    public var showDebugSpokenInstructionsOnMap = false
```

Comment:
My feedback on this property was ignored – https://github.com/mapbox/mapbox-navigation-ios/pull/727#discussion_r145258675 – so here’s some tail work: #756 #757.